### ISSUE TITLE
SeekToPosition method

### ISSUE THREAD

**Issue by DebashishDey** [lipangit/JiaoZiVideoPlayer#2305]
Please, can you help me with the usage of seekToPosition method, can't seem to find how to use it.

**Comment by lipangit** [comment]
![image](https://user-images.githubusercontent.com/2038071/52947645-4832e100-33b2-11e9-8082-1e0cef911aaf.png)

do you mean it?  after setUp  before start

**Comment by DebashishDey** [comment]
> ![image](https://user-images.githubusercontent.com/2038071/52947645-4832e100-33b2-11e9-8082-1e0cef911aaf.png)
> 
> do you mean it? after setUp before start

Yes, but which method to use to seekTo a particular position? wanted to implement skip functionality like youtube?.

**Comment by lipangit** [comment]
Bro no youtube in china, I will never see that function.

not need method, just seekToInAdvance = something .

**Comment by DebashishDey** [comment]
Setting seekToInAdvance = value is not setting the video current time to seekToInAdvance time.

**Comment by lipangit** [comment]
do you want to seek the video position like click progressbar?

**Comment by DebashishDey** [comment]
Yes, but programmatically.

**Comment by lipangit** [comment]
1.you can click progressbar programmatically.
2.JZMediaManager.seekTo(int);   when playing
